src/dnf-plugins/product-id/README.md

```diff
@@ -14,11 +14,11 @@ libdnf and microdnf packages from default repositories or you can use
 following steps:
 
 * VM running Fedora 28.
-* Master branch of libdnf:
+* Main branch of libdnf:
```

Comment:
There is no `main` branch in the libdnf repository.

Reply:
I know it's not. I figured that this word may be just an adjective and not strictly name of the branch. But I understand that it may be confusing because `master` is an established name after all. Reverting.